### PR TITLE
Add option to specify custom links in the web console

### PR DESCRIPTION
**Pull request by glrf**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,4 @@ _archive/
 _public/
 
 # Additional entries
+jsonnetfile.json
```
Comment (simu) [inline .gitignore:25]: I don't think this should be gitignored, as there's a possibility that we'll want to specify Jsonnet dependencies in this component at some point.

**Review by simu — APPROVED** [review]
LGTM overall.

File: .gitignore (hunk 1 above)
Comment (glrf) [reply .gitignore:25]: Fair enough. It just got annoying to not be able to add just `git add .`

Comment (simu) [reply .gitignore:25]: That's what `.git/info/exclude` or a global `gitignore` is for ;)

**Pull request merged**